e2e/app/component/edit-delete-annotations-modal.ts

```diff
@@ -0,0 +1,56 @@
+import {Page} from 'puppeteer';
+import {savePageToFile, takeScreenshot} from 'utils/save-file-utils'
+import {LinkText} from 'app/text-labels';
+import Modal from './modal';
+import {waitForText} from 'utils/waits-utils';
+import BaseElement from 'app/element/base-element';
+
+
+const modalTitle = 'Edit or Delete Review-Wide Annotation Fields';
+
+
+export default class EditDeleteAnnotationsModal extends Modal {
+
+  constructor(page: Page, xpath?: string) {
+    super(page, xpath);
+  }
+
+  async waitForLoad(): Promise<this> {
```

Comment:
are you overriding `waitForLoad` function in `Modal` class intentionally?

Reply:
@aweng98 frankly, I did not give much thought to it! Here I've reused the `new-notebook-modal.ts` file.